public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -822,16 +822,16 @@ export class DashboardMigrator {
 
             // Delete the legacy field
             delete panel.fieldConfig.defaults.custom.displayMode;
+          }
```

Comment:
@codeincarnate pushed a small fix to the migrations, the override migrations where inside the check for the default displayMode (if (displayMode !== undefined) {) , but the default displayMode option can be undefined but overrides can still exist, so moved the override migration outside this if condition. Now the default react table test dashboard works. 

http://localhost:3000/d/U_bZIMRMk/panel-tests-react-table?orgId=1

Reply:
Thanks Torkel! 🙏  I've tested out using the above gdev dashboards and it appears to be working on my end now as well.